6-AdvancedScenarios/4-hybrid-SPA/README.md

```diff
@@ -0,0 +1,412 @@
+      * Select the **Add permissions** button at the bottom.
+1. In the app's registration screen, select the **Expose an API** blade to the left to open the page where you can declare the parameters to expose this app as an API for which client applications can obtain [access tokens](https://aka.ms/access-tokens) for.
+The first thing that we need to do is to declare the unique [resource](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) URI that the clients will be using to obtain access tokens for this API. To declare an resource URI (Application ID URI), follow the following steps:
+   * Select `Set` next to the **Application ID URI** to generate a URI that is unique for this app.
+   * For this sample, accept the proposed Application ID URI (`api://{clientId}`) by selecting **Save**. Read more about Application ID URI at [Validation differences by supported account types \(signInAudience\)](https://docs.microsoft.com/azure/active-directory/develop/supported-accounts-validation).
+1. All APIs have to publish a minimum of one [scope](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code), also called [Delegated Permissions](https://docs.microsoft.com/azure/active-directory/develop/v2-permissions-and-consent#permission-types), for the client's to obtain an access token successfully. To publish a scope, follow these steps:
+   * Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
+        * For **Scope name**, use `access_as_user`.
+        * Select **Admins and users** options for **Who can consent?**.
+        * For **Admin consent display name** type `Allow the app msal-hybrid-spa to [ex, read ToDo list items] as the signed-in user`.
+        * For **Admin consent description** type `Allow the application to [ex, read ToDo list items] as the signed-in user.`
+        * For **User consent display name** type `[ex, Read ToDo list items] as yourself`.
+        * For **User consent description** type `Allow the application to [ex, Read ToDo list items] as the signed-in user on your behalf.`
+        * Keep **State** as **Enabled**.
+        * Select the **Add scope** button on the bottom to save this scope.
+1. Select the `Manifest` blade on the left.
+   * Set `accessTokenAcceptedVersion` property to **2**.
+   * Set the `optionalClaims` property as shown below to request client capabilities claim `idToken`:
+   ```json
+   "optionalClaims": {
+        "idToken": [
+            {
+                "name": "sid",
+                "source": null,
+                "essential": false,
+                "additionalProperties": []
+            },
+            {
+                "name": "login_hint",
+                "source": null,
+                "essential": false,
+                "additionalProperties": []
+            }
+        ],
+        "accessToken": [],
+        "saml2Token": []
+    }
+   ```
```

Comment:
There is a blade for this now. Lets replace this with new set of instructions generated from codegen